qa/suites/rados/objectstore/backends/objectstore-bluestore-a.yaml

```diff
@@ -8,5 +8,5 @@ tasks:
 - install:
 - exec:
     client.0:
-      - mkdir $TESTDIR/archive/ostest && cd $TESTDIR/archive/ostest && ulimit -Sn 16384 && CEPH_ARGS="--no-log-to-stderr --log-file $TESTDIR/archive/ceph_test_objectstore.log --debug-bluestore 20" ceph_test_objectstore --gtest_filter=*/1:-*SyntheticMatrixC* --gtest_catch_exceptions=0
+      - mkdir $TESTDIR/archive/ostest && cd $TESTDIR/archive/ostest && ulimit -Sn 16384 && CEPH_ARGS="--no-log-to-stderr --log-file $TESTDIR/archive/ceph_test_objectstore.log --debug-bluestore 20" ceph_test_objectstore --gtest_filter=*/1:-*Matrix*Compression* --gtest_catch_exceptions=0
```

Comment:
IMO new filter will filter out most of matrix tests (except ones ending with /1) which is different from the original behavior.

So the proper filter should be:
--gtest_filter=*/1:*Matrix*:-*Matrix*Compression*

Reply:
RIGHT!

Reply:
Right, fixed.